debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -207,13 +207,15 @@
         },
         hide_toolbar: function(setCookie) {
             // close any sub panels
-            $('#djDebugWindow').hide();
+            $('#djDebugWindow').attr('hidden', 'hidden').hide();
             // close all panels
-            $('.djdt-panelContent').hide();
+            $('.djdt-panelContent').attr('hidden', 'hidden').hide();
             $('#djDebugToolbar li').removeClass('djdt-active');
             // finally close toolbar
-            $('#djDebugToolbar').hide('fast');
-            $('#djDebugToolbarHandle').show();
+            $('#djDebugToolbar').hide('fast', function() {
+                $(this).attr('hidden', 'hidden')
+            });
```

Comment:
How come you decided to use the callback function to remove the hidden attribute here rather than following the same method for all the other changes?